zebra/zebra_vty.c

```diff
@@ -389,6 +390,10 @@ static void vty_show_ip_route(struct vty *vty, struct route_node *rn,
 		if (CHECK_FLAG(re->flags, ZEBRA_FLAG_SELECTED))
 			json_object_boolean_true_add(json_route, "selected");
 
+		if (dest->selected_fib == re)
```

Comment:
No Null check for dest ? 

I saw NULL check present in few other places where rib_dest_from_rnode() is called.

Reply:
if you have a re then you have a dest.  Impossible to get here otherwise.